### PR TITLE
Escape the brackets in sanction and quarantine docs

### PR DESCRIPTION
**Pull request by webbushka**
The non escaped brackets are breaking the dev-portal. Need to have them escaped for the docusaurus renderer

## Summary by CodeRabbit

- **Documentation**
  - Improved formatting for event attribute values in the quarantine and sanction specifications for better readability and alignment.

### CODE REVIEW

**Review by scirner22 — DISMISSED** [review]

File: x/quarantine/spec/04_events.md
Comment (SpicyLemon) [inline x/quarantine/spec/04_events.md:20]: This is incorrect syntax for markdown tables. the dashes are supposed to go all the way between the pipes.

File: x/quarantine/spec/04_events.md
Comment (SpicyLemon) [inline x/quarantine/spec/04_events.md:10]: These TOCs are auto-generated using `make update-tocs`. This change will be undone the next time someone runs that.

**Review by Taztingo — DISMISSED** [review]

File: x/quarantine/spec/04_events.md
Comment (webbushka) [reply x/quarantine/spec/04_events.md:20]: The formatter I have in VS Code automatically did that. I've copied and pasted it here... seems like it works like that.

| Attribute Key | Attribute Value                        |
| ------------- | -------------------------------------- |
| to_address    | \{bech32 string of account opting in\} |

File: x/quarantine/spec/04_events.md
Comment (webbushka) [reply x/quarantine/spec/04_events.md:10]: No worries, the change was automatically made by the formatter in vscode

**Comment by webbushka** [comment]
I don't have auth to merge these after they are approved if someone is willing to merge that would be appreciated.

@scirner22 @Taztingo @SpicyLemon

File: x/quarantine/spec/04_events.md
Comment (SpicyLemon) [reply x/quarantine/spec/04_events.md:20]: A lot of markdown interpreters understand it, but the actual syntax definition says to have them go all the way across, so it's not guaranteed to work.

File: x/quarantine/spec/04_events.md
Comment (SpicyLemon) [reply x/quarantine/spec/04_events.md:10]: This change needs to be undone before we merge. Otherwise, it'll just get included in some other unrelated PR when some spec documentation is updated.

File: x/quarantine/spec/04_events.md
Comment (webbushka) [reply x/quarantine/spec/04_events.md]: Updated

File: x/quarantine/spec/04_events.md
Comment (webbushka) [reply x/quarantine/spec/04_events.md]: Updated

**Review by Taztingo — APPROVED** [review]

**Review by SpicyLemon — APPROVED** [review]

**Comment by SpicyLemon** [comment]
Looks like there's an unsigned commit: `ad2c5e4` `@webbushka Merge branch 'aj/docs-escape-brackets' of github.com:provenance-io/provenance into aj/docs-escape-brackets`

**Comment by SpicyLemon** [comment]
superseded by #2007

**Pull request closed without merging**
